tools/build_test_env.sh

```diff
@@ -30,7 +30,7 @@ NOVENV=
 PY_VER=3
 API_VER=4
 GITLAB_IMAGE="gitlab/gitlab-ce"
-GITLAB_TAG="latest"
+GITLAB_TAG="nightly"
```

Comment:
I guess this could hurt, if GitLab changes one of their APIs around a couple times.

Let's stay at stable and wait for the 22nd

Reply:
It was temporary to test on Travis, `latest` doesn't ship the API yet.

Reply:
I hope the new release will ship the feature; there're some bugs to be fixed: https://gitlab.com/gitlab-org/gitlab/-/issues/211963, https://gitlab.com/gitlab-org/gitlab/-/issues/211878

Reply:
I started #1054 as a first step to try and improve on this problem, so it can still be tested. We will need proper skippable test cases (based on gitlab versions) to make this smoother, hopefully very soon :P.